modules/search-solr-client/src/main/scala/io/renku/search/solr/query/SolrToken.scala

```diff
@@ -73,8 +73,7 @@ object SolrToken:
   def dateLt(field: Field, date: Instant): SolrToken =
     fieldIs(field, s"[* TO ${fromInstant(date)}]")
 
-  // TODO: currently only projects work, user can't be decoded
-  val allTypes: SolrToken = fieldIs(Field.Type, "Project")
+  val allTypes: SolrToken = Entity.allTypes.map(fieldIs(Field.Type, _)).toList.combineAll
```

Comment:
What do you think - can we say `Field.Type, "*"` maybe? It might be simpler/faster than a combination of `or`?

Reply:
Sure, good idea.